### ISSUE TITLE
Non-truncated repr for generators

### ISSUE THREAD

**Issue by victorjzsun** [alexmojaki/cheap_repr#20]
Currently, generators default to `object` type since `generator` cannot be found in `repr_registry`, but it uses a differently named `__repr__` (`<slot wrapper '__repr__' of 'generator' objects>`). Similar to #15, this results in a truncated repr.
```python
def this_is_a_long_function():
    generator = (2 * n for n in range(50))
    print(repr(generator))
    print(cheap_repr(generator))

this_is_a_long_function()
```
Returns
```
<generator object this_is_a_long_function.<locals>.<genexpr> at 0x7f8599a5c900>
<generator object this_is_a_....<genexpr> at 0x7f8599a5c900>
```

**Comment by laike9m** [comment]
@alexmojaki Kindly ping

**Comment by alexmojaki** [comment]
Fixed in https://github.com/alexmojaki/cheap_repr/commit/9d2d6435b92a00bcd0e128987f267258818caa25, released in 0.4.6

**Comment by laike9m** [comment]
Thanks a lot!
